build.gradle.kts

```diff
@@ -0,0 +1,73 @@
+import org.gradle.api.tasks.testing.logging.TestExceptionFormat
+import org.gradle.api.tasks.testing.logging.TestLogEvent
+
+plugins {
+    id("java-gradle-plugin")
+    id("com.gradle.plugin-publish") version "0.12.0"
+    groovy //TODO remove it at the end of the migration
+    id("com.adarshr.test-logger") version "2.1.1" //TODO latest version incompatible with existing groovy version
+}
+
+java {
+    sourceCompatibility = JavaVersion.VERSION_1_8
+    targetCompatibility = JavaVersion.VERSION_1_8
+}
+
+repositories {
+    mavenCentral()
+    gradlePluginPortal()
+}
+
+dependencies {
+    implementation("io.gatling:gatling-enterprise-plugin-commons:1.5.5") {
+        exclude("com.fasterxml.jackson.core")
```

Comment:
awkward but maybe that's how gradle works.  gatling-enterprise-plugin-commons directly depends on jackson-databind. Shouldn't you use constraints instead? https://docs.gradle.org/current/userguide/dependency_constraints.html